docs/release_notes/version_chart/versions_support.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Prophecy versions support
+id: versions_support
+description: Prophecy versions support
+sidebar_position: 1
+tags: [compatibility, matrix, version, chart, library, plib, plibs]
+---
+
+This versions support page describes the Prophecy version system, version types, and version lifecycles.
```

Comment:
```suggestion
This page describes the Prophecy version system, version types, and version lifecycles.
```
i feel like saying versions so many times is redundant but fine if you keep it too